components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -886,6 +886,30 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
                     return;
                 }
+
+                // (AT) At this point we found a running/building prebuild, which might also include
+                // image build in current state.
+                //
+                // The owner's client connection is automatically registered to listen on instance updates.
+                // For the remaining client connections which would handle `createWorkspace` and end up here, it
+                // also would be reasonable to listen on the instance updates of a running prebuild, or image build.
```

Comment:
+++
We need to be forwarded the WorkspaceInstanceUpdates in the frontend, because we do not have any other means to reliably learn about the status about image builds, yet. Once we have those, we should remove this.